src/planwise/client/mapping.cljs

```diff
@@ -45,7 +45,7 @@
 ;; only "classic" mapbox styles can be used with this static API.
 ;; the new API does not allow defining an overlay as a query parameter.
 (defn static-image [geojson]
-  (fmt/format "https://api.mapbox.com/v4/%s/geojson(%s)/auto/256x144.png?access_token=%s"
+  (fmt/format "https://api.mapbox.com/v4/%s/geojson(%s)/auto/256x158.png?access_token=%s"
```

Comment:
If we remove this value from the CSS, then the project cards change size when the image is actually loaded. 

In order to de-duplicate the value, I looked for some way to inject a config value into SASS as a variable but found nothing. Another option would be to keep the definition of a `$map-preview-image-size` variable in an `scss` file, and load it and parse it from clojure code to extract the value, and pass it onto the client via `config`; but it feels overkill.